ntoskrnl/io/iomgr/file.c

```diff
@@ -917,7 +917,10 @@ IopParseDevice(IN PVOID ParseObject,
                 /* Make sure the file object knows it has an extension */
                 FileObject->Flags |= FO_FILE_OBJECT_HAS_EXTENSION;
 
+                /* Initialize file object extension */
```

Comment:
Cool find!
The correct fix should be done on line 860:
There should be `RtlZeroMemory(FileObject, ObjectSize);`

Reply:
Done @Extravert-ir. Updated the PR. I can confirm that your suggestion also fixes the problem for me, same as my previous changes. :slightly_smiling_face: